wallet.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package rusk;
+option go_package = "github.com/dusk-network/rusk-schema;rusk";
+
+message EmptyRequest {}
+
+message BalanceResponse {
+	fixed64 unlockedBalance = 1;
+	fixed64 lockedBalance = 2;
+}
+
+service WalletService {
```

Comment:
If there are no name clashes, the preferred naming for this would be `service Wallet`, to avoid unnecessary stuttering.

Reply:
A few services are declared like this (BidService, StakeService) maybe we can update them with this or another PR too?

Reply:
Definitely not in this PR, since it is out of scope. In these cases, there are structures that take up the name (`Bid`, `Stake`). So in this case, it is okay to stutter in the service now. If it is not necessary however, it's best to avoid it.